app/src/main/java/com/myfitnesstracker/ui/main/MainViewModel.kt

```diff
@@ -17,16 +17,16 @@ import com.myfitnesstracker.ui.dto.BMI
 
 class MainViewModel : ViewModel() {
 
-   private var mainFragment: MainFragment = MainFragment()
+   private var mainFragment : MainFragment = MainFragment()
    //private var storageReference = FirebaseStorage.getInstance().reference
-   private var _bmis: MutableLiveData<ArrayList<BMI>> = MutableLiveData<ArrayList<BMI>>()
+   private var _bmis : MutableLiveData<ArrayList<BMI>> = MutableLiveData<ArrayList<BMI>>()
    private var _exercises = MutableLiveData<ArrayList<ExerciseDTO>>()
    private var _nutrition = MutableLiveData<ArrayList<Nutrition>>()
    private var _nutritionSearch = MutableLiveData<ArrayList<NutritionSearchResultDTO>>()
-   private lateinit var firestore: FirebaseFirestore
-   var exerciseService: ExerciseService = ExerciseService()
-   var nutritionService: NutritionService = NutritionService()
-   var nutritionSearchResultService: NutritionSearchService = NutritionSearchService()
+   private lateinit var firestore : FirebaseFirestore
+   private var exerciseService : ExerciseService = ExerciseService()
+   private var nutritionService : NutritionService = NutritionService()
+   private var nutritionSearchResultService : NutritionSearchService = NutritionSearchService()
```

Comment:
Added Visibility Modifiers according to: https://kotlinlang.org/docs/reference/visibility-modifiers.html